content/python/concepts/built-in-functions/terms/sum/sum.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: 'sum()'
+Description: 'Takes in an iterable object, such as a list or tuple, and returns the sum of all elements.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Functions'
+  - 'Iterators'
+  - 'Lists'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+---
+
+Takes in an iterable object, such as a list or tuple, and returns the sum of all elements.
+
+## Syntax
+
+```python
+sum(iterable, start=0)
+```
+
```

Comment:
We would like a sentence after the syntax code block explaining what `iterable` and `start` are.

Reply:
I have added the following:

**iterable:** is an object capable of returning its elements one at a time
for example a list, tuple or dictionary.

**start:** is a number added to the sum of the numbers in the iterable.